tests/Core/Content/Product/SalesChannel/Listing/ProductListingFeaturesSubscriberTest.php

```diff
@@ -574,15 +580,19 @@ private function initMocks(): void
             $sorting->setKey('score');
             $sorting->setFields(['score' => ['field' => '_score', 'order' => 'asc']]);
             $sorting->setUniqueIdentifier('score');
-            $productSorting = new ProductSortingCollection([$sorting]);
-            $this->productListingSortingRegistry->expects($this->any())
-                ->method('getProductSortingEntities')
-                ->willReturn($productSorting);
+            $this->sortingCollection = new ProductSortingCollection([$sorting]);
+
+            if (method_exists($this->productListingSortingRegistry, 'getProductSortingEntities')) {
```

Comment:
Shouldn't the version check ensure that the method does exist?

Reply:
I prefer not do do a version check if I can do it without one. A simple `method_exists` is sufficient here.

Reply:
This version check is not really related.

* The class `ProductListingSortingRegistry` exists since Shopware 6.3.2.
* The method `ProductListingSortingRegistry::getProductSortingEntities` was removed with Shopware 6.4.